ledger-service/http-json/src/main/scala/com/digitalasset/http/WebSocketService.scala

```diff
@@ -527,18 +607,24 @@ class WebSocketService(
       jwt: Jwt,
       jwtPayload: JwtPayload,
   )(implicit
-      lc: LoggingContextOf[InstanceUUID with RequestID]
+      ec: ExecutionContext,
+      lc: LoggingContextOf[InstanceUUID],
+      _jwt: Jwt = jwt,
```

Comment:
Why are there two `Jwt`s here, one of which is `implicit`?

Reply:
got rid of the implict jwt. Better pass it explicit.